modules/pruning-deployments.adoc

```diff
@@ -41,13 +41,15 @@ To also prune replica sets associated with `Deployment` objects, use the `--repl
 .^|`--orphans`
 |Prune all replication controllers that no longer have a `DeploymentConfig` object, has status of `Complete` or `Failed`, and has a replica count of zero.
 
+ifndef::openshift-rosa,openshift-dedicated[]
```

Comment:
Not sure if these are working as intended, I see this note in all previews (OCP, ROSA, and OSD).

Reply:
@kcarmichael08 The ifdefs are working; but the preview isn't updating. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/2a6eda16-90fd-4d86-8435-46e0172e4bf4)